client/flags/flags.go

```diff
@@ -45,7 +45,7 @@ const (
 	FlagUseLedger        = "ledger"
 	FlagChainID          = "chain-id"
 	FlagNode             = "node"
-	FlagGRPC             = "grpc"
+	FlagGRPC             = "grpc-addr"
```

Comment:
It is actually interesting that it reads from `.simapp/config/app.toml` if it is only `grpc`.
I want to avoid putting `grpc.address` otherwise the value will be set by default and it will always create a grpc client.